qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -608,7 +608,7 @@
     <item>
      <widget class="QComboBox" name="comboBox_5">
       <property name="sizePolicy">
-       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
```

Comment:
[`QSizePolicy::Preferred`](http://doc.qt.io/qt-5/qsizepolicy.html#Policy-enum) means:

> The sizeHint() is best, but the widget can be shrunk and still be useful. The widget can be expanded, but there is no advantage to it being larger than sizeHint()

[`QComboBox`](http://doc.qt.io/qt-5/qcombobox.html#sizeAdjustPolicy-prop) by default chooses its size hint to fit its contents the first time it's shown. Though, I believe the size that you see in the after screenshot is the smallest size hint it will naturally choose.

> So if there's a control inside it that can resize as large as it likes I'm not sure how those end up settling.

The size policies are a little brain-fuzzling, but a widget that actually wants to resize to be larger will have a size policy which includes the `ExpandFlag`, like [`QSizePolicy::Expanding`](http://doc.qt.io/qt-5/qsizepolicy.html#Policy-enum). I *think* if you put an expanding widget as a child of a preferred size widget, the child will fill any extra space within the parent, but will not change the size of the parent.